src/compiler/builderState.ts

```diff
@@ -36,8 +36,7 @@ import {
 } from "./_namespaces/ts";
 
 /** @internal */
-export function getFileEmitOutput(program: Program, sourceFile: SourceFile, emitOnlyDtsFiles: boolean,
-    cancellationToken?: CancellationToken, customTransformers?: CustomTransformers, forceDtsEmit?: boolean): EmitOutput {
+export function getFileEmitOutput(program: Program, sourceFile: SourceFile, emitOnlyDtsFiles: boolean, cancellationToken?: CancellationToken, customTransformers?: CustomTransformers, forceDtsEmit?: boolean): EmitOutput {

```

Comment:
Is this because of line limit? after changing limit will it make parameters on new line?

Reply:
It's because there are only two lines instead of strictly one line per parameter or one single line for the entire signature.

There might be a dprint setting for this sort of thing (preferHanging?) but it won't work unless we set our line limit lower.

Reply:
I think if the setting was to move to multiline that would be better